### PR TITLE
Updates to the README

### PR DESCRIPTION
**Pull request by legalosLOTR**
# Description
Trying to better organize the readme and convey all the important stuff. This is just the first draft so happy to hear feedback :)
Also, there are a bunch of tbd's which we can continue to add over time.

# Safety checklist
* [x] This change is backwards compatible and safe to apply by existing users
* [x] This change will NOT lead to data loss
* [x] This change will NOT lead to downtime who already has an env/service setup

## How has this change been tested, beside unit tests?
N/A

### CODE REVIEW

**Review by juandiegopalomino — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
   
 </p>
 
-# Try out Opta
+# Features
```
Comment (legalosLOTR) [inline README.md:56]: Add a why opta section before this

File: README.md [hunk 2]
```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
...
+
+### Cloud services
+Opta supports the 3 major clouds - AWS, GCP and Azure. It has modules for the most commonly used services in these clouds like:
+* Microservices (powered by [Kubernetes](https://docs.opta.dev/architecture/aws/))
+* Databases - Postgres, MySQL, Redis
+* Serverless workloads
+* Networking - VPCs, Subnets, Load balancers
+* Cloudfront
```
Comment (RemyDeWolf) [inline README.md:64]: this list of features is not cloud specific so we could say Content Delivery Network (CDN)

File: README.md [hunk 3]
```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
...
+* Object storage (S3, GCS)
 
+Additionally, we bake in best practices like:
+* Observability (Datadog, LogDNA)
```
Comment (RemyDeWolf) [inline README.md:68]: link is https://docs.opta.dev/observability/

File: README.md [hunk 4]
```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
...
+* [SOC2 compliance](https://docs.opta.dev/compliance/)
+* [Continuous Deployment](https://docs.opta.dev/tutorials/continuous_deployment/)
+* [Hardened network and security configurations](https://docs.opta.dev/architecture/aws/)
```
Comment (RemyDeWolf) [inline README.md:71]: may be decline in 3 links: aws/gcp/azure

File: README.md [hunk 5]
```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
...
 
-To use Opta, you first need to create some simple yaml configuration files that describe your needs. You can use:
 
-* Our [magical GUI](https://app.runx.dev/yaml-generator) to help generate these files.
+### Coexistence with existing Infrastructure
+Opta aims to be compatible with your existing Infrastructure setup. You can:
 
-* The [getting started](https://docs.opta.dev/getting-started/) guide and the detailed docs.
+* Import existing Terraform infrastructure into Opta
+* Use Opta outputs in Terraform files
```
Comment (RemyDeWolf) [inline README.md:79]: do you mean the ability to generate TF files?  -> WIP

File: README.md [hunk 6]
```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
...
+* Write [custom Terraform modules](https://docs.opta.dev/reference/aws/environment_modules/custom-terraform/) (for services that Opta doesn't support yet)
+* Run Opta in existing VPCs (WIP)
+
+# Try out Opta
 
-* Also, check out more [examples](https://github.com/run-x/opta/tree/main/examples)
+The best place to get started is the [Getting Started Guide](https://docs.opta.dev/getting-started/).
```
Comment (RemyDeWolf) [inline README.md:85]: get started - Getting Started -> Read the [Getting Started Guide]

File: README.md [hunk 7]
```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
...
 
 # Development
-[Dev guide](https://github.com/run-x/opta/blob/main/development.md)
+We love user contributions! Check out our [Dev guide](https://github.com/run-x/opta/blob/main/development.md) to get started.
+
+# Miscellaneous
+* [Team behind Opta](https://www.runx.dev/about)
+* How Opta delivers upgrades (WIP)
+* Bugfix / Feature request policy (WIP)
+* Comparision with other tools (WIP)
```
Comment (RemyDeWolf) [inline README.md:96]: Comparison

File: README.md [hunk 8]
```diff
@@ -55,27 +53,46 @@ If you'd like to try it out or have any questions - feel free to join our [Slack
...
+* Public roadmap (WIP)
+* Case studies (WIP)
```
Comment (RemyDeWolf) [inline README.md:98]: what about [Fyre](https://blog.flyte.org/how-opta-makes-deploying-flyte-much-easier)?

File: README.md (hunk 6 above)
Comment (legalosLOTR) [reply README.md:85]: Can you elaborate? I don't understand what needs to be changed.

Comment (RemyDeWolf) [reply README.md:85]: not a big deal, but we can avoid the repetition of get started 2x

**Review by RemyDeWolf — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (legalosLOTR) [reply README.md]: ah fixed!

**Pull request merged**